packages/artillery/lib/util/await-on-ee.js

```diff
@@ -1,19 +1,21 @@
 const sleep = require('./sleep');
 
-async function awaitOnEE(ee, message, pollMs = 1000) {
+async function awaitOnEE(ee, message, pollMs = 1000, maxWaitMs = Infinity) {
```

Comment:
change made here to allow breaking the while loop. kept it backward compatible by introducing the default, so the current behaviour is not changed for other consumers of the function